### ISSUE TITLE
Update Detroit Primary Color Value for Accessibility

### ISSUE THREAD

**Issue by slowbot** [CityOfDetroit/bloom#979]
What feature is this part of?
Accessibility updates

Is there any context that will help with completing this task?
Multiple color contrast errors are produced from the use of a primary color with insufficient contrast
https://github.com/CityOfDetroit/bloom/issues/925
https://github.com/CityOfDetroit/bloom/issues/926
https://github.com/CityOfDetroit/bloom/issues/927
https://github.com/CityOfDetroit/bloom/issues/928

Are there any potential sub-steps necessary to complete this task?
Update the tailwind config file with the new primary color #297E73
https://github.com/CityOfDetroit/bloom/blob/main/sites/public/tailwind.config.js

Review overwrites file for uses of recent primary color change

What is the definition of done?
Use Wave browser tool and test for color contrast and get not errors for small text AA

**Comment by slowbot** [comment]
LGTM @kathyccheng @jaredcwhite
